VirtoCommerce.Storefront/Controllers/Api/ApiListsController.cs

```diff
@@ -141,7 +142,7 @@ public async Task<ActionResult> ClearList(string listName, string type)
         // POST: storefrontapi/lists/search
         [HttpPost("search")]
         [ValidateAntiForgeryToken]
-        public async Task<ActionResult<ShoppingCartSearchResult>> SearchLists([FromBody] CartSearchCriteria searchCriteria)
+        public async Task<ActionResult<IList<ShoppingCart>>> SearchLists([FromBody] CartSearchCriteria searchCriteria)
```

Comment:
You should return ShoppingCartSearchResult, public contracts cannot not be broken.

Reply:
done